CONTRIBUTING.md

```diff
@@ -232,6 +232,36 @@ the event happens.
 Make log message human readable and also include data that is needed for easier
 understanding of what happened and in what context.
 
+### Observability
+
+Out of the box, your users should be able to observe the state of your component. 
+The collector exposes a `/metrics` endpoint where your metrics will land.
```

Comment:
Prometheus `/metrics` endpoint?

Reply:
OpenMetrics would be more accurate, I suppose.